spec.md

```diff
@@ -451,7 +451,7 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id?acc
 ### Response
 
 | Status Code | Description |
-|---|---|
+| --- | --- |
```

Comment:
No idea what the correct format is but this was the only one that didn't have them so I decided this one was the odd man out.  :-)
There's this doc:  https://help.github.com/articles/organizing-information-with-tables/    which shows it with spaces.